cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -900,7 +900,7 @@
       public static final String CACHE_TTL_SECS = "security.authorization.cache.ttl.secs";
       /** Maximum number of entries the authorization cache will hold */
       public static final String CACHE_MAX_ENTRIES = "security.authorization.cache.max.entries";
-      public static final String ADMIN_USERS = "security.authorization.admin.users";
```

Comment:
But removing it from here does not resolve security loophole or backward compatibility correct ? We can expect users to specify new groups who will have admin but what about admin.users who have already been granted access. Do we need to drop privileges for them ?